samples/wifi/scan/README.rst

```diff
@@ -20,9 +20,20 @@ Overview
 ********
 
 This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.
+The default scan type is active scan.
+However, you can configure the scan type to be either active or passive.
 
 Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.
 
```

Comment:
You might want to add here the Configuration heading. See the sample doc template: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html#configuration

Reply:
Yeah, added, please review

Reply:
Not quite yet there. Proposing to have:

Configuration
***************
|config|

note about the scan type

Configuration options
==============

I believe :kconfig:option:`CONFIG_WIFI_SCAN_TYPE_SELECT` should work here, as the new option is going to be available in the Kconfig reference, right?

Reply:
The parsing went a bit wrong in my comment, but I hope you get the idea. Just like in the template.

Reply:
Sorry for stealing the review from you @kittydepa. Please continue from here.